doc/dev/installation.rst

```diff
@@ -411,6 +411,6 @@ To generate html files for the documentation for Catalyst:
 
 .. code-block:: console
 
-  pip install -r doc/requirements.txt
+  make docs
```

Comment:
I think you do not to install the requirements first though

Reply:
Installing requirement has already been at llne 385.